### PR TITLE
Update install_pfft.sh

### PR DESCRIPTION
**Pull request by HerculesJack**
Added a condition check for failure of the curl command

### CODE REVIEW

File: depends/install_pfft.sh [hunk 1]
```diff
@@ -20,6 +20,18 @@ curl -L -o $ROOT/depends/pfft-$PFFT_VERSION.tar.gz \
 https://github.com/rainwoodman/pfft/releases/download/$PFFT_VERSION/pfft-$PFFT_VERSION.tar.gz
 fi
 
+if ! [ -f $ROOT/depends/pfft-$PFFT_VERSION.tar.gz ]; then
+echo wget -P $ROOT/depends/ \
```
Comment (rainwoodman) [inline depends/install_pfft.sh]: did you want echo here?

Comment (HerculesJack) [reply depends/install_pfft.sh]: I didn't... Now fixed.

**Comment by rainwoodman** [comment]
Could you do a forced push -- to trigger the ci? I forgot to enable CI on pull requests.

**Comment by HerculesJack** [comment]
Did you want me to do "$ git push -f origin master" ? I did and it said "Everything up-to-date".

**Comment by rainwoodman** [comment]
Probably need to make a new commit with git commit --amend?

**Comment by HerculesJack** [comment]
OK. I've done that.

**Comment by rainwoodman** [comment]
Yes. Travis is green.

**Pull request merged**